static/docs/commands-reference/commit.md

```diff
@@ -2,40 +2,212 @@
 
 Record changes to the repository by saving outputs to cache.
 
+## Synopsis
+
 ```usage
     usage: dvc commit [-h] [-q | -v] [-f] [-d] [-R] [targets [targets ...]]
 
     positional arguments:
       targets               DVC files.
-
-    optional arguments:
-      -f, --force           Commit even if checksums for dependencies/outputs
-                            changed.
-      -d, --with-deps       Commit all dependencies of the specified target.
-      -R, --recursive       Commit cache for subdirectories of the specified
-                            directory.
 ```
 
+## Description
+
+Normally DVC commands like `dvc add`, `dvc repro` or `dvc run`, commit the data
+to the DVC cache as the last step. What _commit_ means is that DVC
+
+* Computes a checksum for the file
```

Comment:
Could we clarify that this works for the directory as well?

Reply:
I would add a sentence saying "For a directory a checksum is computed representing the directory as a whole, and not for the individual files" ??? I'm uncertain what the checksum value that's computed represents. 

It's not useful to get down in the weeds.  There's a JSON file that's inserted into the cache listing the files in the directory, but we don't need to mention that in the document.

Reply:
What I ended up with:  "Computes a checksum for the file, or for the contents of a directory 
  structure."